pandas/tseries/tdi.py

```diff
@@ -269,7 +269,7 @@ def _simple_new(cls, values, name=None, freq=None, **kwargs):
     @property
     def _formatter_func(self):
         from pandas.formats.format import _get_format_timedelta64
-        return _get_format_timedelta64(self, box=True)
+        return _get_format_timedelta64(self.values, box=True)
```

Comment:
why are you changing this? these routines can and are supposed to take actual objects, not ``.values``